src/examples/logrotate

```diff
@@ -7,6 +7,6 @@
     compress
     delaycompress
     postrotate
-        /bin/kill -HUP `cat /var/run/sssd.pid  2>/dev/null`  2> /dev/null || true
+        test -r /var/run/sssd.pid && /bin/kill -HUP `cat /var/run/sssd.pid 2>/dev/null` &>/dev/null || true
     endscript
```

Comment:
I think, this is unnecessary

In the report they have

    kill -HUP `cat /var/run/sssd.pid` > /dev/null 2>&1 || true

and we have

    /bin/kill -HUP `cat /var/run/sssd.pid  2>/dev/null`  2> /dev/null || true

We suppress the error message already. Ubuntu should use what we have here. I do not see real benefit in
testing the existence of sssd.pid file here. In my opinion it is just less readable.

Reply:
I agree this seems unnecessary, but the github ticket that you can find in the description complains exactly about that. That is what seems to work differently in Ubuntu than Fedora.

That said, I asked the reporter @AlexisWilke to provide more information.

Reply:
Well, I got more information I summarized in [this comment](https://github.com/SSSD/sssd/issues/6741#issuecomment-1814436111), and this changes where unnecessary. I removed them.